backend/pkg/modules/internal/bfla/bfladetector/learn_and_detect_bfla.go

```diff
@@ -88,9 +88,9 @@ type BFLADetector interface {
 	ApproveTrace(path, method string, clientRef *k8straceannotator.K8sObjectRef, apiID uint, user *DetectedUser)
 	DenyTrace(path, method string, clientRef *k8straceannotator.K8sObjectRef, apiID uint, user *DetectedUser)
```

Comment:
shall we return the error also for Approve/Deny/Update?

Reply:
let's keep this in a separate discussion